### DISCUSSION TITLE
Set HOME via nsswitch.conf

### DISCUSSION THREAD

**Discussion by bradenmcd** [git-for-windows/git#3947] [Q&A] [answered]
Has anyone had any luck setting `HOME` via `nsswitch.conf` in a Git for Windows installation? From what I could find in the online docs for Cygwin/MSYS, this appears to be the official way to do it; but when I try setting `db_home` to a path in Git for Windows' `/etc/nsswitch.conf`, the setting seems to be completely ignored. (In my case, `HOME` seems like it's getting derived from existing `HOMEDRIVE`/`HOMEPATH` environment variables, even if the path is all that's listed for `db_home`.)

**Comment by rimrul** [comment, marked as answer]
`/etc/nsswitch.conf` is probably read after `HOME` gets set in `git-wrapper.c`, and then cygwin/msys2 seems to prefer the existing value instead of what `/etc/nsswitch.conf` says, [according to their FAQ](http://www.cygwin.com/faq.html#faq.setup.home)

**Reply by bradenmcd** [reply]
Aha. Thanks!
